src/ocean/LibVersion.d

```diff
@@ -0,0 +1,9 @@
+module ocean.LibVersion;
+
+const has_features_2_6 = true;
```

Comment:
Wait! Since previous versions won't have the `hasFeaturesFrom` (or even the features definition constants), does it make sense to define the features for old versions?